fideslog/api/database/csv_writer.py

```diff
@@ -0,0 +1,37 @@
+import csv
+from uuid import uuid1
+
+from ..schemas.analytics_event import AnalyticsEvent
```

Comment:
is there no other way to import from fideslog? relative imports can get spooky, in my opinion at least!

Reply:
Updated this one! I'll open up a follow up issue and throw it in the next sprint to do the rest (there are quite a few of them)

Reply:
Opened #102 and added to next sprint